expr/func_aspercent_test.go

```diff
@@ -0,0 +1,544 @@
+package expr
+
+import (
+	"math"
+	"strconv"
+	"testing"
+
+	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/test"
+	"gopkg.in/raintank/schema.v1"
+)
+
+func TestAsPercentSingleNoArg(t *testing.T) {
+
+	out := []schema.Point{
+		{Val: math.NaN(), Ts: 10},
+		{Val: math.NaN(), Ts: 20},
+		{Val: 100, Ts: 30},
+		{Val: math.NaN(), Ts: 40},
+		{Val: math.NaN(), Ts: 50},
+		{Val: 100, Ts: 60},
+	}
+	testAsPercent(
+		"single-noarg",
+		[]models.Series{
+			{
+				Interval:   10,
+				QueryPatt:  "func(tag='something';tag2='anything')",
```

Comment:
Why are there single quotes around the tag values throughout this file?

Reply:
changed